holochain-introduction-for-developers/holochain-brief-introduction.md

```diff
@@ -0,0 +1,79 @@
+# What is Holochain?
+
+> Holochain is a _protocol_ and _framework_ for creating cryptographically secure, portable, and distributed applications. It provides the necessary tools to allow individuals to maintain control over their identity and data, while interacting safely with a network of untrusted peers.
+>
+> The purpose of this document is to help developers and technologists quickly understand:
+>
+> * Why we're building Holochain
+> * What the project consists of
+> * What advantages it has over server-based and blockchain-based architectures
+
+## The problem
+
+The Internet is decentralized by design. Many services, such as the web and email, have no central points. But in order to move through the Internet confidently, we need **trust.** How can you know I am who I say I am, and how can you be sure that I'm acting with integrity?
+
+Centralized services act as proxies of trust. We may not be able to trust a million anonymous peers, but we can trust these big services to faithfully negotiate our interactions with those peers — **until we can't.** These systems are beginning to show weaknesses that are direct consequences of centralization.
+
+* **They are more attractive to attackers.**
+* **They leave users vulnerable to service outages.**
+* **They concentrate power in the hands of a few.**
+* **They burden the service operator with liabilities and costs.**
+
+Blockchain opened our imaginations to the possibility of trustable, decentralized coordination with untrusted peers, but suffers from efficiency problems that are inherent to its design. The next wave of decentralized ledgers improve performance and resilience but carry over some of blockchain's assumptions. They hint at the full potential of this technology but stop before they reach the finish line.
+
+## Holochain: empowered, distributed, trustable cooperation
+
+Holochain gives you the tools to create applications that enable direct interactions among mutually sovereign peers. It provides integrity guarantees without central servers or a mining class. We are building:
+
+* A **peer-to-peer protocol** that shares data among participants, giving them the power to act collectively as an 'immune system' that detects invalid data and pushes out bad actors.
+* An open-source **application development framework** that abstracts away complexity and allows you to focus on your core functionality.
+* A set of **core apps and utilities**, including an identity manager, a private data vault, and a software distribution platform.
+* A **distributed hosting ecosystem** called Holo Hosting which bridges Holochain to the web so people can use your apps as if they were traditional, server-based apps.
+
+### Self-determination
```

Comment:
@zippy in this section I'm trying to focus specifically on the agency of the agent in language that's easy to understand. 'Agency' is a good word but I feel like its common meaning isn't clear enough for use here. Does "Self-determination" feel better here?

You'll note that I also added a section just after this that shows Holochain is not meant for isolated individuals; it's meant for social interaction more than anything. Do you think that these two together is adequate for this short sales pitch? I think that, to do real justice to the concept of mutual sovereignty, I'd have to expand this into a full treatise on voice/exit, shared agreements, power dynamics, etc. Not the place for it ;)